### ISSUE TITLE
[BUG] Screen overflows horizontally in mobile view.

### ISSUE THREAD

**Issue by Joshua-Nweze** [One-Language/Website#117]

**Comment by jbampton** [comment]
Do you have a screenshot ?

**Comment by Joshua-Nweze** [comment]
Yes, I do. When swiped to the left or right the Page overflows horizontally when swiped to the left or right.

![Screenshot_20221005-223623](https://user-images.githubusercontent.com/101072509/194218198-5673da7a-1c61-4a77-9c6e-ed09bb92f95c.jpg)
![Screenshot_20221005-223558](https://user-images.githubusercontent.com/101072509/194218205-4364b23f-4d87-4bbf-96a6-517e5775d556.jpg)

**Comment by jbampton** [comment]
I tried to reproduce the issue in Chrome and Firefox on Android but could not.

When I swipe left or right the page does not move.

What browser and version are you using ?

**Comment by Joshua-Nweze** [comment]
Google for mobile

**Comment by Vishrut19** [comment]
@jbampton I can't see this issue on my Mac. Can you please send the screenshot if this issue is still there ?
<img width="1470" alt="Screenshot 2022-10-09 at 11 46 35 PM" src="https://user-images.githubusercontent.com/53470896/194772950-34febcfa-d93f-45a1-bc02-c7561929b641.png">

**Comment by Joshua-Nweze** [comment]
This is a screen record of the web page in Google Chrome

[screen-capture.webm](https://user-images.githubusercontent.com/101072509/194797094-687f3ec3-877f-42ad-ae79-c87224293122.webm)

**Comment by david-marko** [comment]
Hello,
I have cloned and tested this on different screen sizes and not seeing the overflow issue as indicated in the video
